instrumentation/reactor-netty/reactor-netty-1.0/javaagent/src/test/groovy/AbstractReactorNettyHttpClientTest.groovy

```diff
@@ -179,4 +188,40 @@ abstract class AbstractReactorNettyHttpClientTest extends HttpClientTest<HttpCli
     assert expectedSpanContext.traceId == actualSpanContext.traceId
     assert expectedSpanContext.spanId == actualSpanContext.spanId
   }
+
+  // custom address resolver that returns at most one address for each host
```

Comment:
```suggestion
  // custom address resolver that returns at most one address for each host
  // (because reactor-netty will retry on connection failure across multiple resolved addresses
  // which creates spans that are not expected by the http client tests)
```